python-avd/pyavd/_eos_designs/structured_config/network_services/ethernet_interfaces.py

```diff
@@ -75,6 +75,14 @@ def ethernet_interfaces(self: AvdStructuredConfigNetworkServices) -> list | None
                                 "flow_tracker": self.shared_utils.get_flow_tracker(l3_interface, "l3_interfaces"),
                             }
 
+                            if get(self._l3_interface_acls, self.shared_utils.hostname):
+                                interface.update(
+                                    {
+                                        "access_group_in": get(self._l3_interface_acls[self.shared_utils.hostname], f"{interface_name}.ipv4_acl_in.name"),
+                                        "access_group_out": get(self._l3_interface_acls[self.shared_utils.hostname], f"{interface_name}.ipv4_acl_out.name"),
```

Comment:
Preventing issue with `.` in the interface name. Make sure this is also done anywhere else where you use `get()` with the interface names (for code changed in this PR)
```suggestion
                                        "access_group_in": get(self._l3_interface_acls[self.shared_utils.hostname], f"{interface_name}..ipv4_acl_in..name", separator=".."),
                                        "access_group_out": get(self._l3_interface_acls[self.shared_utils.hostname], f"{interface_name}..ipv4_acl_out..name", separator=".."),
```

Reply:
Fixed.